be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -700,8 +701,13 @@ Status FragmentExecutor::execute(ExecEnv* exec_env) {
     {
         SCOPED_TIMER(prepare_instance_timer);
         SCOPED_TIMER(prepare_driver_timer);
-        RETURN_IF_ERROR(_fragment_ctx->iterate_drivers(
-                [state = _fragment_ctx->runtime_state()](const DriverPtr& driver) { return driver->prepare(state); }));
+        Status st;
+        TRY_CATCH_ALL(st,
```

Comment:
why we need catch exception here?

Reply:
what if here has exceptions? it will kill the process.

Reply:
prepare method just return error status when it fails.
no need to catch exception.